test/unit/models/test_proposals.py

```diff
@@ -277,3 +277,13 @@ def test_approved_and_ranked(go_list_proposals):
 
     assert prop_list[0].object_hash == u'dfd7d63979c0b62456b63d5fc5306dbec451180adee85876cbf5b28c69d1a86c'
     assert prop_list[1].object_hash == u'0523445762025b2e01a2cd34f1d10f4816cf26ee1796167e5b029901e5873630'
+
+
+def test_proposal_size(proposal):
+    orig = Proposal(**proposal.get_dict())  # make a copy
+
+    # fixture as-is should be valid
+    assert proposal.is_valid() is True
+
+    proposal.url = 'https://' + ('x' * 513) + '.com/'
```

Comment:
This looks a bit unrelated tbh :) `'x' * 512` would also pass, no?

Reply:
Yes. The intent was to ensure the total proposal size is over the limit.

I'm not sure what's not related -- what would you suggest changing?

Reply:
Shouldn't it test smth like:
- 512 bytes proposal - ok
- 513 bytes proposal - fail

The way it's written now there is no explicit test for that 512 (MAX_DATA_SIZE) bytes threshold.

Reply:
Good catch! Tightening the test caught a bug where `>=` should have been `>`.